### DISCUSSION TITLE
failed start new session -- display monitor operation

### DISCUSSION THREAD

**Discussion by smasa19** [SafeExamBrowser/seb-win-refactoring#658] [General]
seb caught unexpected exception while performing operation displaymonitoroperation, please help.

[image](https://imgur.com/a/RNPvUEK)

OS: windows 10 pro, version 21H2 (Build OS 19044.2364)
fyi, this laptop has dual graphic, Intel & Nvidia

and this the log.

[2023-09-07_12h14m43s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/12544984/2023-09-07_12h14m43s_Runtime.log)

[2023-09-07_12h14m43s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/12544983/2023-09-07_12h14m43s_Client.log)

**Comment by dbuechel** [comment]
There appears to be something wrong with the .NET installation on your system, given the exception:

```
Exception Message: Could not load file or assembly 'System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
Exception Type: System.IO.FileNotFoundException
```

Try to repair or re-install the .NET Framework (for SEB 3.5.0, you'd at least need version 4.7.2).

**Reply by smasa19** [reply]
ok, ty before for the answer, i've to wait that student bring those laptop again, see if i can resolve the issue, i'll post either it work or not,
greet from java
